backend/migrations/versions/4c9e3982a64b_create_user_theatrical_role_.py

```diff
@@ -0,0 +1,45 @@
+"""create user_theatrical_role_relationship table
+
+Revision ID: 4c9e3982a64b
+Revises: 4ef6789fa0fc
+Create Date: 2023-12-16 01:46:26.170771
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = "4c9e3982a64b"
+down_revision: Union[str, None] = "4ef6789fa0fc"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "user_theatrical_role_relationships",
+        sa.Column("id", sa.Uuid(), nullable=False),
+        sa.Column("user_id", sa.Uuid(), nullable=False),
+        sa.Column("theatrical_role_id", sa.Uuid(), nullable=False),
+        sa.ForeignKeyConstraint(
+            ["theatrical_role_id"],
+            ["theatrical_roles.id"],
+        ),
+        sa.ForeignKeyConstraint(
+            ["user_id"],
+            ["users.id"],
+        ),
+        sa.PrimaryKeyConstraint("id"),
```

Comment:
By the way, I don't know how much data we will have in the database, but it's worth to create index on ForeingKey `user_id`

Reply:
because this is a table of relationships there is a composite index on the user-role relationship